.ci/scripts/release/post-github.sh

```diff
@@ -1,6 +1,11 @@
-#!/bin/bash
+#!/bin/bash -xue
 
-cd "clients/go/internal/embedded"
+# Publish Go tag for the release
+echo git tag "clients/go/v${RELEASE_VERSION}"
```

Comment:
```suggestion
git tag "clients/go/v${RELEASE_VERSION}"
```

I assume you actually want to execute the git command?